run_tests.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
```

Comment:
This file is new. It's the pattern I've been following for testing our third party apps that have not aged well.